README-source.md

```diff
@@ -616,6 +616,9 @@ const getList = (z, bundle) => {
 };
 ```
 
+**`bundle.meta.zap.id` is only available in the `performSubscribe` and `performUnsubscribe` methods**
+
+The user's Zap ID is available during the [subscribe and unsubscribe](https://github.com/zapier/zapier-platform-schema/blob/master/docs/build/schema.md#basichookoperationschema) methods.
 
```

Comment:
Is it possible to add a small code example on how one could use `bundle.meta.zap.id` from `performSubscribe`?